youi/src/player/KalturaVideoPlayer.cpp

```diff
@@ -132,6 +145,13 @@ void KalturaVideoPlayer::ChangePlaybackRate(float playbackRate)
     PlaybackRateChangedEvent.Emit(playbackRate);
 }
 
+void KalturaVideoPlayer::SetPlayerZIndex(float zIndex)
+{
+    YI_LOGD(TAG, "SetPlayerZIndex %f", zIndex);
+    m_pPriv->SetPlayerZIndex_(zIndex);
+    PlayerZIndexChangedEvent.Emit(zIndex);
```

Comment:
as this is coming from the app it doesn't need notification on the change, so we can remove this and also the other code that handles event dispatching